pkg/inline/inline_test.go

```diff
@@ -116,9 +131,11 @@ func TestInlineBundle(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error parsing bundle: %v", err)
 	}
-	inliner := &Inliner{
-		Reader: &fakeLocalReader{},
-	}
+	inliner := NewInlinerWithScheme(
```

Comment:
This test is hard to read because the test data is in a different place than the test.  Can you (1) break the test up into smaller chunks and (2) inline (lol) the test data with the test so it's easier to read (input, output)?